tests/Forc.toml

```diff
@@ -0,0 +1,9 @@
+[project]
+author = "Nicholas Furfaro"
+entry = "main.sw"
+license = "Apache-2.0"
+name = "tests"
+
+[dependencies]
+core = { git = "http://github.com/FuelLabs/sway-lib-core" }
+std = { git = "http://github.com/FuelLabs/sway-lib-std" }
```

Comment:
Should this not be a local path, so you can test the _current_ stdlib?

Reply:
hmm. How will that play with CI though? I feel like using a local path while developing, then pushing to github and using "latest"(as in the example above)  might be what we need here?

Reply:
Using local `std` and git's `core` seems to make the most sense here, and would pass in CI since it will use the currently-being-tested version of `std`, right?